jekyll/_cci2/windows-images-support-policy.adoc

```diff
@@ -0,0 +1,107 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= CircleCI Windows images support policy
+:page-layout: classic-docs
+:page-liquid:
+:page-description: CircleCI Windows images release, update, and deprecation policy
+:icons: font
+:experimental:
+
+[#overview]
+== Overview
+
+This document outlines the CircleCI Windows image release, update, and deprecation policy. This policy applies to all Windows images built for the Windows Executor.
+
+[#release-policy]
+== Release policy
+
+The Windows images are based on Windows Server. We aim to maintain and release two major versions of Windows Server at all times. Currently we support `Windows server 2019` and `Windows server 2022`
+
+Tags will be redirected to the updated images automatically as described in the xref:#tagging[tagging] section of this document. We will announce these releases on our link:https://discuss.circleci.com/[Discuss Forum].
+
+[#tagging]
+== Tagging
+
+We support various tags for the Windows environment, allowing you to choose a suitable image for your needs.
+
+- `default`: This image is the current stable image. Jobs will default to this if no image is specified.
+
+- `current`: This version of Windows will receive updates approximately every 3 months. If no tag is provided, `current` will be the version that jobs default to.
+
+- `edge`: This tag is reserved for previews of new releases, which will initially point to this tag. The tag may include incremental updates relative to the current quarterly image release, which may change without notice, and is not recommended for production CI workloads. The `current` tag is promoted from `edge`
```

Comment:
tags are reserved for previews of new releases, which will initially point to this tag. The edge tags may include incremental updates to the current quarterly image release, which may change without notice, and is not recommended to be used for production CI workloads. currentwill be updated with these changes after period of stability.

Reply:
will do